pyproject.toml

```diff
@@ -7,6 +7,7 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.6"
+matplotlib = "^3.3.4"
```

Comment:
is this necessary over here? If not, we can move it to dev dependencies

Reply:
It's to make sure that the notebook runs well. But moving it to dev dependencies also sounds good, I'll make the changes

Reply:
Moving it to the main dependency means the user requires this to install matplotlib when using pydp. This is not desirable behavior.

Reply:
I've already moved matplotlib as a dev dependency